src/components/plot-container.js

```diff
@@ -133,24 +148,41 @@ export default function PlotContainerFactory(MapContainer) {
     render() {
       const {exportImageSetting, mapFields, splitMaps} = this.props;
       const {imageSize = {}, legend} = exportImageSetting;
+      const {mapState} = mapFields;
       const isSplit = splitMaps && splitMaps.length > 1;
 
       const size = {
         width: imageSize.imageW || 1,
         height: imageSize.imageH || 1
       };
+
+      const bounds = findMapBounds(mapFields.visState.layers);
+      const width = size.width / (isSplit ? 2 : 1);
+      const height = size.height;
       const scale = this.mapScaleSelector(this.props);
+      const newMapState = {
+        ...mapState,
+        width,
+        height,
+        zoom: mapState.zoom + (Math.log2(scale) || 0)
```

Comment:
nit: instantiate zoom value above and reuse here and in line 177.

Reply:
this and line 177 shouldn't be reading the same zoom value because on line 167 we read the current map zoom value and on line 177 we are recomputing zoom and center based on data layers middle point

Reply:
Ah you are right. Might be worth wrapping it in a function then (feel free to ignore).